NN-notebook-v7/notebook-v7.md

```diff
@@ -0,0 +1,118 @@
+
+These three concerns will drive our transition; their specific implications are detailed below in the form of concrete user stories.
+
+## Extensions
+
+The Notebook ecosystem for extensions is rich and varied, and some of those are critical to major deployments, especially in educational and institutional contexts. Since the internal APIs of JupyterLab are entirely different from those of Notebook v6, existing extensions written for v6 will not work on v7 out of the box. Here we detail the plan for how to minimize the disruption caused by this transition and how to support the community in the process.
+
+There is a [community repository of unofficial Notebook extensions](https://jupyter-contrib-nbextensions.readthedocs.io), along with [corresponding documentation](https://jupyterlab-contrib.github.io/migrate_from_classical.html) indicating which of these extensions have analogs or ports in JupyterLab. Some of the functionalities provided by these unofficial Notebook extensions are now built-in features in JupyterLab: code folding, collapsible headings, the keyboard shortcuts editor, the table of contents, and many others.
+
+### Critical extensions needed for Notebook v7
+
+The following Notebook v6 extensions have been flagged by the community as critical to several major deployments and widely-used workflows. We will work with the extension developers to help port these (or help make new equivalent extensions) for Notebook v7:
+
+- [Nbgrader](https://github.com/jupyter/nbgrader)
+- [RISE](https://github.com/damianavila/RISE)
+- [Jupytext](https://github.com/mwouts/jupytext/issues/271)
+- IPython Parallel (filebrowser tab)
+
+### Extensions whose functionality will be built into Notebook v7
+
+The following extensions will have largely equivalent experiences shipping in core Notebook v7. We will need to track whether there are any remaining differences in their functionality that are either regressions, or potentially disrupting or confusing to existing users.
+
+- [Table of Contents](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/toc2)
+- [Collapsible headings](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/collapsible_headings)
+
+## Major new features in Notebook v7
+
+In addition to having parity with the Notebook v6 user experience and widely-used extensions, Notebook v7 will inherit major new features that have been developed in the JupyterLab project, thus providing automatically a number of improvements to Notebook users as they transition to v7:
+
+- Debugger
+- Real-time collaboration
+- Theming and dark mode
+- Internationalization
+- Improved Accessibility
```

Comment:
out of all the new features, accessibility is the least developed. i'm worried that if miscommunicate these features we might again distance ourselves from the disability community. i think i'm curious about how we make sure accessibility becomes a feature.

Reply:
```suggestion
- Improved Web Content Accessibility Guideline compliance
```